x/wasm/client/cli/query.go

```diff
@@ -46,7 +48,9 @@ func NewQueryCmd(cdc *codec.CodecProxy, reg codectypes.InterfaceRegistry) *cobra
 		NewCmdGetContractState(cdc, reg),
 		NewCmdListPinnedCode(cdc, reg),
 		GetCmdLibVersion(cdc, reg),
+		NewCmdListContractBlockedMethod(cdc),
 		NewCMDParams(cdc, reg),
+		NewCMDGetAddressWhiteList(cdc, reg),
```

Comment:
Use `NewCmdXXX` to be consistent with other commands

Reply:
done